django-stubs/core/mail/message.pyi

```diff
@@ -66,12 +63,8 @@ class SafeMIMEMultipart(MIMEMixin, MIMEMultipart):  # type: ignore
     def __setitem__(self, name: str, val: str) -> None: ...
 
 _AttachmentContent: TypeAlias = bytes | EmailMessage | Message | SafeMIMEText | str
-# switch to tuple once https://github.com/python/mypy/issues/11098 is fixed
-# remove Optional once python 3.7 is dropped (Tuple[str | None, ...] works with mypy on py3.10)
 _AttachmentTuple: TypeAlias = (
-    Tuple[str, _AttachmentContent]
-    | Tuple[Optional[str], _AttachmentContent, str]
-    | Tuple[str, _AttachmentContent, None]
+    tuple[str, _AttachmentContent] | tuple[str | None, _AttachmentContent, str] | tuple[str, _AttachmentContent, None]
```

Comment:
Hrm? These comments were referencing Python 3.7, but I'm not sure whether this has anything to do with Python 3.7.

`str | None` or `tuple[str]` don't work on Python 3.8 runtime either.

But AFAIK it doesn't matter since `.pyi` files are only evaluated by mypy, not Python itself.

Reply:
These files are parsed by python, but not executed. I don't remember the origin of this comment, but I am happy that it is removed now :)

Reply:
Yes, exactly. `mypy` uses `ast.parse` to get the Python's AST and then `mypy` builds its own AST around it.